meta/style.pm

```diff
@@ -607,10 +607,6 @@ sub CheckInOutParams
     my $ptr = $4;
     my $param = $5;
 
-    if ($type eq "sai_object_id_t" and not $line =~ /_In_ sai_object_id_t \w+|_In_ const sai_object_id_t \*\w+|_Out_ sai_object_id_t \*\w+/)
```

Comment:
Please revert this back

Reply:
this change will affect all objects, and for multiple ojects use bulk functions

Reply:
ok